common/src/test/java/org/apache/comet/parquet/TestFileReader.java

```diff
@@ -615,12 +617,12 @@ public void testColumnIndexReadWrite() throws Exception {
   @Test
   public void testWriteReadMergeScanRange() throws Throwable {
     Configuration conf = new Configuration();
-    conf.set(ReadOptions.COMET_IO_MERGE_RANGES, Boolean.toString(true));
+    conf.set(CometConf.COMET_IO_MERGE_RANGES().key(), Boolean.toString(true));
```

Comment:
This test is setting the config values in a Hadoop Configuration still, and not setting them in the Spark config. Would it make sense to update the test?

Reply:
There is no spark context in this test. I've added a new test with the configuration set thru the spark config